src/components/Dashboard.tsx

```diff
@@ -616,6 +600,7 @@ function Dashboard() {
     if(checkPremium() === false){
       return;
     }
+    alert('Thanks for your membership!')
```

Comment:
here you probably want to open your normal UI for when people click on "Add Wallet".

Also, the paywall script sets a value in localstorage so that it "knows" the address of the last connected user. If you need to debug this you probably want to remove that value so that the paywall starts "fresh" :)